MarsRover/Rover.cs

```diff
@@ -17,16 +15,66 @@ public Rover(Direction direction, int x, int y)
 
         public Location Location { get; private set; }
 
-        public Direction Turn(Command command) // TODO: talk at Mentor meeting about changing to switch expression
+        public void ExecuteCommand(Command command)
+        {
+            switch (command)
+            {
+                case Command.TurnLeft:
+                    TurnLeft();
+                    break;
+                case Command.TurnRight:
+                    TurnRight();
+                    break;
+                default:
+                    return;
+            }
+        }
+
+        private void TurnLeft()
         {
+            Direction = Direction switch 
+            {
+                Direction.North => Direction.West,
+                Direction.West => Direction.South,
+                Direction.South => Direction.East,
+                Direction.East => Direction.North,
+                _ => Direction.None
+            };
+        }
+
+        private void TurnRight()
+        {
+            Direction = Direction switch
+            {
+                Direction.North => Direction.East,
+                Direction.West => Direction.North,
+                Direction.South => Direction.West,
+                Direction.East => Direction.South,

```

Comment:
bit nit picky but it could be easier to read if the previous result was the new case i.e. North, East, South, West map to East, South, West, North respectively